user-guide/Advanced_Modules/Application_framework/Application_framework.md

```diff
@@ -4,17 +4,12 @@ uid: Application_framework
 
 # DataMiner Low-Code Apps
 
-With the DataMiner Low-Code Apps, you can create custom applications that interact with data from a DataMiner System or an external source. Available from DataMiner 10.2.5/10.3.0 onwards.
+With the DataMiner Low-Code Apps module, you can create custom applications that interact with data from a DataMiner System or from an external source.
 
-- [Accessing custom applications](xref:Accessing_custom_apps)
+The DataMiner Low-Code Apps module is fully available from DataMiner 10.2.5/10.3.0 onwards. Prior to this, starting from DataMiner 10.0.8, you can preview it if the *Applications* [soft-launch option](https://community.dataminer.services/documentation/soft-launch-options/) is enabled.
 
-- [Creating custom applications](xref:Creating_custom_apps)
-
-- [Editing a custom application](xref:Editing_custom_apps)
-
-- [Configuring custom applications](xref:Configuring_custom_apps)
-
-- [Deleting a custom application](xref:Deleting_custom_apps)
+> [!IMPORTANT]
+> The Low-Code Apps module requires the "Low-Code Apps" license. For more information on acquiring this license, contact the Skyline Sales department.
 
 > [!NOTE]
-> The Application Low-Code Apps require the "Low-Code Apps" license. For more information on acquiring this license, contact the Skyline Sales department.
+> To view, add, edit, delete, or publish low-code applications, you need to have the necessary user permissions under [Modules > User-definable apps](xref:DataMiner_user_permissions#modules--user-definable-apps) as well as the [DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission.
```

Comment:
Are low-code applications and Low-Code Apps both fine to use? So one with capitals and the other without?

Reply:
Indeed, we use "Low-Code Apps" when referring to the unique module, but low-code application when referring to the non-unique thing created using the module. This is based on the difference between a proper and common noun:
![image](https://user-images.githubusercontent.com/94605575/188067803-c865f1d2-6cef-4e81-a7a9-75878c26549f.png)
(See https://grammar.yourdictionary.com/capitalization/10-rules-of-capitalization.html)